src/averageAges.js

```diff
@@ -14,12 +14,14 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const averageAges = century ? people.filter((human) =>
+    human.sex === 'm' && Math.ceil(human.died / 100) === century)
+    : people.filter((human) => human.sex === 'm');
```

Comment:
Пропоную ось в такому форматі записати
винеси колбеки в окремі змінні
```suggestion
  const averageAges = people.filter(
    century ? firstCallback : secondCallback
  )
```